extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,739 @@
+<!-- omit in toc -->
 # 3DTILES_implicit_tiling
 
 **Version 0.0.0**, December 1, 2020
 
+<!-- omit in toc -->
 ## Contributors
 
 * Sam Suhag, Cesium
 * Sean Lilley, Cesium
 * Peter Gagliardi, Cesium
 * Erixen Cruz, Cesium
+* Patrick Cozzi, Cesium
 
+<!-- omit in toc -->
 ## Status
 
 Draft
 
+<!-- omit in toc -->
 ## Dependencies
 
 Written against the 3D Tiles 1.0 specification.
 
+<!-- omit in toc -->
 ## Contents
 
-- [3DTILES_layers](#3dtiles_layers)
-  - [Contributors](#contributors)
-  - [Status](#status)
-  - [Dependencies](#dependencies)
-  - [Contents](#contents)
-  - [Overview](#overview)
-  - [Schema Changes](#schema-changes)
+- [Overview](#overview)
+- [Use Cases](#use-cases)
+- [Tiling Schemes](#tiling-schemes)
+  - [Implicit Subdivision](#implicit-subdivision)
+- [Tile Coordinates](#tile-coordinates)
+- [Template URIs](#template-uris)
+- [Content](#content)
+- [Subtrees](#subtrees)
+- [Availability](#availability)
+  - [Tile Availability](#tile-availability)
+  - [Content Availability](#content-availability)
+  - [Child Subtree Availability](#child-subtree-availability)
+- [Subtree JSON Files](#subtree-json-files)
+  - [Buffers and Buffer Views](#buffers-and-buffer-views)
+  - [Morton Order](#morton-order)
+  - [Availability Encoding](#availability-encoding)
+- [Tileset JSON](#tileset-json)
+- [Glossary](#glossary)
+- [Examples](#examples)
+  - [Quadtree with four levels](#quadtree-with-four-levels)
+- [Octree Example](#octree-example)
+- [JSON Schema Reference](#json-schema-reference)
+- [Appendix A: Tree Indexing Formulas](#appendix-a-tree-indexing-formulas)
+  - [Morton Indexing Formulas](#morton-indexing-formulas)
+  - [Availability Formulas](#availability-formulas)
+    - [Tile and Content Availability Formulas](#tile-and-content-availability-formulas)
+    - [Child Subtree Availability Formulas](#child-subtree-availability-formulas)
 
-## Contents
-Contents go here.
 ## Overview
 
-This extension to 3D Tiles enables implicit tiling. 
+>**Implicit tiling** is a alternative method for describing a Cesium 3D Tileset that provides a more succinct representation for large tilesets. It encodes the hierarchical structure of the tileset in binary, as opposed to explicit tiling, which specifies the tile hierarchy in JSON. The Cesium 3D Tiles 1.0 specification only supports explicit tiling, as every tile is listed in the tileset JSON file.
+
+Implicit tiling keeps the tileset JSON file small, which makes loading large tilesets faster. While explicit tiling can be used to represent large datasets, the tileset JSON file grows linearly with the number of tiles. Implicit tiling keeps the tileset JSON file bounded in size.
+
+Implicit tiling also provides a method for accessing tiles by tile coordinates. This allows for abbreviated tree traversal algorithms.
+
+![Explicit vs Implicit Tiling](figures/implicit-vs-explicit.jpg)
```

Comment:
Should clarify binary tree is for visualization only since we only support quad and oct.